src/System.Console/tests/TermInfo.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+
+public class TermInfo
+{
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public void VerifyInstalledTermInfosParse()
+    {
+        foreach (string location in ConsolePal.TermInfo.Database._terminfoLocations)
+        {
+            if (!Directory.Exists(location))
+                continue;
+
+            foreach (string term in Directory.EnumerateFiles(location, "*", SearchOption.AllDirectories))
+            {
+                if (term.ToUpper().Contains("README")) continue;
+                ConsolePal.TerminalColorInfo info = new ConsolePal.TerminalColorInfo(ConsolePal.TermInfo.Database.ReadDatabase(Path.GetFileName(term)));
+
+                if (string.IsNullOrEmpty(info.ForegroundFormat) == false)
```

Comment:
I know you prefer the ```== false```... that's fine, we should just be consistent, especially in the same function.... above you're using ```!``` with Directory.Exists.